### ISSUE TITLE
Rule option request: `multiline-ternary` to disallow newlines

### ISSUE THREAD

**Issue by ljharb** [eslint/eslint#6751]
The `multiline-ternary` rule allows me to require ternaries to be multiline - I'd like an option that requires ternaries *not* be multiline.

**What version of ESLint are you using?**
v3.1.1

**What rule do you want to change?**
`multiline-ternary`, I'd like a "disallowMutliline" option added.

**What code should be flagged as incorrect with this change?**
```js
var x = a ?
  b :
  c;

var y = a
  ? b
  : c;

var z = a ? b :
  c;
```

**What happens when the rule is applied to this code now?**
It passes.

**Comment by vitorbal** [comment]
Thanks, @ljharb, I know it can be a bit tedious, but would you mind filling in the [template for proposing a rule change](http://eslint.org/docs/developer-guide/contributing/rule-changes#proposing-a-rule-change)?

**Comment by ljharb** [comment]
@vitorbal ah, sorry, i didn't realize there was a template for that as well. Will update.

**Comment by mysticatea** [comment]
I set JSCS Compatibility milestone since JSCS has [disallowMultiLineTernary](http://jscs.info/rule/disallowMultiLineTernary) rule.

**Comment by kaicataldo** [comment]
Interestingly enough, this was discussed and we ended up deciding to only implement "always". I'm happy to add this in if/when it's accepted.

**Comment by kaicataldo** [comment]
@eslint/eslint-team Thoughts? Looks like we need two more thumbs on this.

**Comment by kaicataldo** [comment]
For context: https://github.com/eslint/eslint/issues/6066#issuecomment-227143246

**Comment by ilyavolodin** [comment]
I'll support this. Seems like a reasonable enhancement to `multiline-ternary`. 👍

**Comment by platinumazure** [comment]
:+1: from me.

**Comment by ilyavolodin** [comment]
@kaicataldo If you are championing this, I think you can mark it as accepted at this point.
